docs/pages/pmd/projectdocs/decisions/adr-2.md

```diff
@@ -0,0 +1,65 @@
+---
+title: ADR 2 - Policy on the use of Kotlin for development
```

Comment:
Maybe we should not make this specifically about Kotlin, but about non-Java languages in general? Or keep it for now limited to Kotlin? Then my example with Scala should be removed...

Reply:
I think this document is well-written and I like the amount of context specifically on kotlin. I would say, we should keep it as is

Reply:
:+1: